cpp/unit/type_pair.cpp

```diff
@@ -0,0 +1,5 @@
+// Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+namespace openage {
+
+} // namespace openage
```

Comment:
you can remove the cpp file if the header is sufficient. otherwise, newline in last line is missing^^